Design/WorkspacePropertyWithIndex.md

```diff
@@ -112,6 +107,9 @@ The simplest scenario for a call to an algorithm could be:
 ChangePulsetime(TimeOffset=10, InputWorkspaceWithIndex=("ws", SpectrumNumber, [1:33, 42]))
 # or 
 ChangePulsetime(TimeOffset=10, InputWorkspaceWithIndex=("ws", SpectrumNumber, "1:33, 42"))
+
+# optionally no indices. Will just assume all spectra 
+ChangePulsetime(TimeOffset=10, InputWorkspaceWithIndex=("ws", SpectrumNumber))
```

Comment:
In that case the argument `SpectrumNumber` should also not be there, I think? And we do not need to pass a tuple.